packages/host/app/components/ai-assistant/panel.gts

```diff
@@ -455,7 +455,6 @@ export default class AiAssistantPanel extends Component<Signature> {
     this.isShowingPastSessions = false;
   }
 
-  @cached
```

Comment:
I'm skiptical that this does what you think it does--`@cached` prevents us from having to spin thru all the members to find the specific user ID every time this property is consumed. it seems like a very reasonable use of the @cached keyword

Reply:
Actually, I do believe you are right. This is not the fix that I expect since I was able to re-create the issue (altho not reliably)

Reply:
I have checked that the syanpse db has up-to-date data, the issue is somewhere in indexing or leakage of the resources map